docker-maintainer/scripts/start.sh

```diff
@@ -87,24 +94,32 @@ then
   sed -i 's/app\\\.php/app\\_dev.php/g' /etc/nginx/sites-enabled/default
   # Run DOMjudge in root
   sed -i '/^\t#location \//,/^\t#\}/ s/\t#/\t/' /etc/nginx/sites-enabled/default
-   sed -i '/^\tlocation \/domjudge/,/^\t\}/ s/^\t/\t#/' /etc/nginx/sites-enabled/default
+  sed -i '/^\tlocation \/domjudge/,/^\t\}/ s/^\t/\t#/' /etc/nginx/sites-enabled/default
+  # Set up permissions (make sure the script does not stop if this fails, as this will happen on macOS / Windows)
```

Comment:
Why is this permission fixing steps necessary? Shouldn't make maintainer-install deal with all that?

Reply:
These steps are normally done in the nginx webserver setup make step I think, which I do not use.